src/core/config/descriptions.h

```diff
@@ -109,4 +112,10 @@ constexpr auto kDInsertStatements = "Rows to be inserted into the table using th
 constexpr auto kDDeleteStatements = "Rows to be deleted from the table using the delete operation";
 constexpr auto kDUpdateStatements = "Rows to be replaced in the table using the update operation";
 constexpr auto kDNDWeight = "Weight of ND to verify (positive integer)";
+constexpr auto kDOcContext = "Context of the OC to verify";
+constexpr auto kDOcLeftIndex = "Index of the left attribute of the OC to verify";
+constexpr auto kDOcRightIndex = "Index of the right attribute of the OC to verify";
+auto const kDODLeftOrdering = details::kDODLeftOrdering;
+constexpr auto kDOFDContext = "Context of the OFD to verify";
```

Comment:
shouldn't it be `details::kDODLeftOrdering.data()` as all the similar lines here that defines `char *`?